### DISCUSSION TITLE
20th open maintainers call

### DISCUSSION THREAD

**Discussion by mxinden** [libp2p/rust-libp2p#3899] [General]
Hi everyone,

The rust-libp2p maintainers will meet tomorrow (Tuesday). See [libp2p calendar](https://calendar.google.com/calendar/embed?src=libp2p.io_0q9682i3te7eanhe9q7ae1c58g%40group.calendar.google.com).

https://meet.jit.si/rust-libp2p-open-maintainers-call

Everyone is welcome to join, listen along, propose topics and join the discussions.

Please post below in case you already want to add an agenda item.

See you all Tomorrow!

**Comment by thomaseizinger** [comment]
I'll be there!

**Comment by thomaseizinger** [comment]
cc @tcoratger @dariusc93 @drHuangMHT 

In case you are interested and haven't seen this. No obligation to join :)

**Comment by dariusc93** [comment]
May I ask what timezome the meetings are usually held at?

**Reply by mxinden** [reply]
> May I ask what timezome the meetings are usually held at?

You should be able to see the reoccurring meeting in the above calendar.

I am currently in Japan, that is why it is 09:00 UTC. Happy to adjust in case this time does not work for you.

**Reply by drHuangMHT** [reply]
I see 2 AM not pm Pacific time? That would be 5 pm for +8, I guess that is fine for me?

**Reply by mxinden** [reply]
Great. See you in a bit then @drHuangMHT.

Yes, it is Europe and Asia friendly. Unfortunately not America friendly, though again, we can alternate in case there is interest from folks in the Americas.
